src/main/java/aeneas/models/PieceFactory.java

```diff
@@ -5,11 +5,286 @@
  * @author Joseph Martin
  */
 public class PieceFactory {
-  static Piece[] pieceIndex = {
-    /* TODO: Fill in pieces here */
+  public static Piece[] pieces = {
```

Comment:
I shall make it so

On Fri, Apr 22, 2016, 01:40 Joseph Martin <notifications@github.com> wrote:

> In src/main/java/aeneas/models/PieceFactory.java
> <https://github.com/Cheddarpuffs/Aeneas-KabaSuji/pull/48#discussion_r60693086>
> :
>
> > @@ -5,11 +5,286 @@
> >   * @author Joseph Martin
> >   */
> >  public class PieceFactory {
> > -  static Piece[] pieceIndex = {
> > -    /* TODO: Fill in pieces here */
> > +  public static Piece[] pieces = {
>
> I don't think this should be public - we don't want these instances being
> given out, since things like rotating pieces will modify them, and if all
> the instances of a particular type of hexonimo refer to the same instances
> from this array, then they'll all change when rotate etc. Instead, I think
> we want to keep the makePiece function, which should clone the piece before
> returning it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Cheddarpuffs/Aeneas-KabaSuji/pull/48/files/021735bfb31687c4cb58986e2da4f22e7e88928c#r60693086>
>